client/app/telecom/telephony/alias/configuration/telecom-telephony-alias-configuration.js

```diff
@@ -8,10 +8,14 @@ angular.module('managerApp').config(($stateProvider) => {
         controllerAs: '$ctrl',
       },
       'featureView@telecom.telephony.alias.configuration': {
-        templateProvider: alias => `<div data-ng-include="::'app/telecom/telephony/alias/configuration/feature/${alias.featureType}/telecom-telephony-alias-configuration-${alias.featureType}.html'"></div>`,
+        templateProvider: (alias) => {
+          const featureType = alias.featureType === 'ddi' ? 'redirect' : alias.featureType;
+          return `<div data-ng-include="::'app/telecom/telephony/alias/configuration/feature/${featureType}/telecom-telephony-alias-configuration-${featureType}.html'"></div>`;
```

Comment:
Could you directly import template and return it ? Maybe something like what has been done here : https://github.com/ovh-ux/manager/pull/71/files#diff-8d871bd6ef1adb3c52908986ae51ee3aR18